charts/prometheus/values.yaml

```diff
@@ -671,6 +671,10 @@ server:
   ## Path to a configuration file on prometheus server container FS
   configPath: /etc/config/prometheus.yml
 
+  ### The data directory used by prometheus to set --storage.tsdb.path
+  ### When empty server.persistentVolume.mountPath is used instead
+  # storagePath: ""
```

Comment:
could you please uncomment the example value?

Reply:
done!